tap_hubspot/sync.py

```diff
@@ -116,11 +117,6 @@ def sync_endpoint(client,
             'sort': sort,
             'latestMessageTimestampAfter': str(start_datetime)
         }
-    
-    if stream_name == 'messages':
-        params = {
-            'limit': str(limit)
-        }
```

Comment:
Would we want to limit the messages stream? I'd imagine that this is guard so that we don't get RateLimitExceptions or other errors on the API side.

Reply:
From my testing, t

1) The messages that belong to a specific thread don't exceed the API pagination limit of 500, and
2) Limiting the message response to 500, doesn't prevent the RateLimit Exception

I am going to add this param value back, to be consistent.